ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml

```diff
@@ -51045,6 +50925,94 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```

Comment:
Above this line looks like it might be a bad commit history... it may be important to create a new PR with a fresh history/base and then add in your changes.